hangman/README.md

```diff
@@ -0,0 +1,96 @@
+# Hangman - Coding Task
+
+## Overview
+**Hangman** is a classic word-guessing game where players attempt to guess a hidden word by suggesting letters. If the player makes too many incorrect guesses, they lose the game.
+
+## Task Description
+The goal is to implement a **text-based** version of Hangman. The game should:
```

Comment:
Do we want to add a link to Wikipedia for reference? https://en.wikipedia.org/wiki/Hangman_(game)